[Optimization][dlink-doc]metadata_table_property value字段过长导致数据插入失败

### Search before asking

- [X] I had searched in the [issues](https://github.com/DataLinkDC/dlink/issues?q=is%3Aissue) and found no similar optimization requirement.


### Description

value字段过长导致数据插入失败（长字段如hudi schema定义）。

```
Caused by: java.sql.BatchUpdateException: Data truncation: Data too long for column 'value' at row 1
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
```

修改metadata_table_property表value字段默认长度为MEDIUMTEXT

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

--- a/dlink-doc/sql/dinky.sql
+++ b/dlink-doc/sql/dinky.sql
@@ -762,7 +762,7 @@ CREATE TABLE `metadata_table` (
 DROP TABLE IF EXISTS `metadata_table_property`;
 CREATE TABLE `metadata_table_property` (
   `key` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_general_ci NOT NULL COMMENT 'key',
-  `value` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_general_ci DEFAULT NULL COMMENT 'value',
+  `value` MEDIUMTEXT CHARACTER SET utf8mb4 COLLATE utf8mb4_general_ci DEFAULT NULL COMMENT 'value',
   `table_id` int NOT NULL COMMENT 'table id',
   `update_time` datetime DEFAULT NULL COMMENT 'update time',
   `create_time` datetime NOT NULL DEFAULT CURRENT_TIMESTAMP COMMENT 'create tiime',

